app/src/main/java/com/lightricks/feedexercise/ui/feed/FeedViewModel.kt

```diff
@@ -97,23 +97,38 @@ open class FeedViewModel () : ViewModel() {
             feedItems = null,
             isLoading = false)
     }
+
+    private fun saveToDb(){
+        val DB_NAME = "user_project_db.db" //TODO: verify
+        val db = Room.databaseBuilder(getApplication(),FeedDatabase::class.java,DB_NAME).build()
+        val userProjectDao = db.userProjectDao()
+        val disposable = userProjectDao.insertAll(feedLstToDbLst(stateInternal.value?.feedItems!!))
+            .subscribeOn(Schedulers.io()).subscribe({},{ error -> handleNetworkError(error)}) //TODO inline the function call or keep as is for readability?
```

Comment:
Longs lines are harder to read. Let's break this one at the dot.

Reply:
This is not a network error. 
Let's rename `handleNetworkError` to something more general.

Reply:
I would put 
`val dbList = feedLstToDbLst(stateInternal.value?.feedItems!!) `
to a separate line.